checklists/intern.json

```diff
@@ -0,0 +1,250 @@
+{"checklistName": "18F New Hire Checklist",
```

Comment:
This should probably say `"18F New Intern Checklist"` or something along those lines.